src/use/use.py

```diff
@@ -520,8 +532,8 @@ def isclass(x):
         return inspect.isclass(x)
 
     @staticmethod
-    def _get_platform_tags() -> set:
-        return set(get_supported()) | set(packaging.tags._platform_tags())
+    def _get_platform_tags() -> Set[str]:
+        return set(get_supported())
```

Comment:
what happened to packaging.tags._platform_tags?

Reply:
it's been moved up into get_supported() because I started using that in one of the tests, not realizing there was a Use._get_platform_tags()